client/src/components/Formik/Input.js

```diff
@@ -39,7 +39,13 @@ const Input = ({ label, name, ...rest }) => {
               margin="normal"
               {...rest}
               {...field}
-              helperText={form.touched[name] ? form.errors[name] : ""}
+              helperText={
+                form.touched[name]
+                  ? form.errors[name]
+                  : helperText
+                  ? helperText
+                  : ""
```

Comment:
Pulling out some nested ternary statements into variables could make this more readable